package.json

```diff
@@ -81,11 +81,20 @@
     ]
   },
   "dependencies": {
+    "@angular/animations": "^8.0.0",
```

Comment:
Do we need these Angular packages in both the dependencies and devDependencies now?

Reply:
Absolutely not! Good catch